sparse/_compressed/compressed.py

```diff
@@ -382,6 +383,12 @@ def tocoo(self):
             .transpose(order)
         )
 
+    def tocsr(self):
+        return CSR(self)
+
+    def tocsc(self):
+        return CSC(self)
+
```

Comment:
I'd prefer for these to not exist, we could direct users to `asformat` instead.

Reply:
:+1: I wasn't sure how many of the construction methods to support.